src/Commands/stubs/console.stub

```diff
@@ -23,19 +23,14 @@ class DummyClass extends Command
 
     /**
      * Execute the console command.
-     *
-     * @return mixed
      */
-    public function handle()
+    public function handle(): void
```

Comment:
I don't think we should add the typehint here for now.

Reply:
I'm okay with this, as `laravel/framework` also does not define the `void` return type in the console `handle()` method, but they do omit the return type. I would suggest we update the docs then too, to match this.